src/app/zap-templates/templates/app/callback.zapt

```diff
@@ -172,9 +174,9 @@ void emberAf{{asUpperCamelCase label}}ClusterClientTickCallback(chip::EndpointId
 /**
  * @brief {{parent.name}} Cluster {{name}} Command callback (from {{source}})
  */
-bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback(chip::EndpointId endpoint,
-       {{#if (isServer source)}}chip::app::CommandSender{{else}}chip::app::CommandHandler{{/if}} * commandObj
-       {{#zcl_command_arguments}}, {{asUnderlyingZclType type}} {{asSymbol label}}{{/zcl_command_arguments}});
+bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback({{#if (isServer source)}}chip::app::CommandSender{{else}}chip::app::CommandHandler{{/if}} * commandObj,
+      const chip::app::ConcreteCommandPath & commandPath, chip::EndpointId endpoint,
```

Comment:
Is there a specific reason for keeping `chip::EndpointId endpoint` as an argument is there is already a `ConcreteCommandPath` ?

Reply:
> Is there a specific reason for keeping `chip::EndpointId endpoint` as an argument is there is already a `ConcreteCommandPath` ?

Boris's answer from slack: "The idea is to incrementally migrate the cluster impls over to the ConcreteCommandPath and the fields arg and then remove the endpoint and the individual command args"

Reply:
Yes, its' to keep scope down and not have lots of changes to clusters all at once that would make review a pain.

There will be followups to eliminate use of the `endpoint` arg and other followups to move from using the individual command args to using the fields struct, then another mass-change to remove all args except the command handler, path, and fields.